cmd/create/machinepool/cmd.go

```diff
@@ -502,7 +568,7 @@ func Split(r rune) bool {
 	return r == '=' || r == ':'
 }
 
-func minReplicaValidator(multiAZ bool) interactive.Validator {
+func minReplicaValidator(multiAZCluster bool, multiAZMachinePool bool) interactive.Validator {
```

Comment:
just send the `multiAZMachinePool` boolean, and don't change the func logic
also for all validator funcs